release_notes/topics/12_0_0.adoc

```diff
@@ -41,6 +41,13 @@ There are also design changes in login theme for better user experience.
 You can even define an icon for your custom Identity providers.
 For details, please refer to the link:{developerguide_link}#custom-identity-providers-icons[docs].
 
+== Default roles processing improvement
+
+Default roles are now internally stored as composite roles of a new role usually named 'default-roles'. Instead of assigning 
```

Comment:
```suggestion
Default roles are now internally stored as composite roles of a new role usually named `default-roles`. Instead of assigning 
```
+ add realm name